[WIP] Update to pwasm-std 0.13

Robbepop opened the pull request:
This updated `pwasm-test` to using `pwasm-std` version 0.11 (later `0.12`).

Makes it possible to drop the `uint` dependency entirely.
Also requires an update to `pwasm-ethereum` - blocking on merging that one first.

Should release a version `0.2` after merging this.

Depends on update to `pwasm-std` for version `0.12`: https://github.com/paritytech/pwasm-std/pull/79

Commit:
Update rust-toolchain from 2018-06-28 to 2018-10-30

Commit:
Update to pwasm-std 0.11

- Also update pwasm-ethereum to updated variant
- Remove uint dependency

dvdplm commented on src/builder.rs:
```diff
@@ -2,8 +2,8 @@ use std::collections::HashMap;
-use pwasm_std::hash::{H256, Address};
-use uint::U256;
+use pwasm_std::types::{H160, H256, U256};
+type Address = H160;
```
Out of curiosity, why isn't `Address` exported from `pwasm-std` (or `pwasm-ethereum`)?

fckt replied:
the same question 🙂. I see a trade-off here also. If we would declare it everywhere that's a bit better for explicitness. But still I'd prefer to declare the alias once somewhere (in `pwasm-std` or `pwasm-abi`)..

Robbepop replied:
This is due to the design decision I have made in version `0.11` of `pwasm-std`.
It made `H160` and `Address` both distinct types which I now regret.
This is going to be reverted in an upcoming version `0.12` of `pwasm-std` because it broke way too much code.

dvdplm commented on tests/calls.rs:
```diff
@@ -2,7 +2,8 @@ extern crate pwasm_test;
 extern crate pwasm_ethereum;
 
-use pwasm_std::hash::Address;
+use pwasm_std::types::{H160};
```
`use pwasm_std::types::H160;`

Robbepop replied:
yep, will do that

dvdplm commented on tests/calls.rs:
```diff
@@ -12,46 +13,46 @@ fn has_called() {
 	let mut result = [0u8; 1];
 	let input = [2u8; 32];
-	pwasm_ethereum::call(2000, &Address::new(), 10000.into(), &input, &mut result).expect_err("Should be an Error");
+	pwasm_ethereum::call(2000, &Address::zero(), 10000.into(), &input, &mut result).expect_err("Should be an Error");
```
I know this boat already sailed, but personally I think `Address::new()` reads better than `Address::zero()`. :/

Robbepop replied:
I am not even the person that declared it being deprecated in version `0.2` of `fixed-hash`, however, I am still in favor of this change since it is more clear what the value of the constructed `fixed-hash` will be.

dvdplm reviewed (approved):
lgtm; I'm not a fan of the `::zero()` name, but maybe it's too late to change that?

Commit:
Update rust-toolchain to 2018-11-12

Commit:
Update to pwasm-std version 0.13

Depends on github dependencies of
- pwasm-abi
- pwasm-ethereum

Those need to be merged into crates.io first.

Commit:
Update to pwasm-ethereum version 0.8

Commit:
Update pwasm-abi dependency to 0.2
